motoko/hello/src/hello_assets/src/index.html

```diff
@@ -16,6 +16,23 @@
         <input id="name" alt="Name" type="text" />
         <button type="submit">Click Me!</button>
       </form>
+      <div id="loader">
+        Please Wait
+        <div class="lds-default">
+          <div></div>
```

Comment:
why all the empty divs? could you possibly accomplish the intent here with css?

Reply:
those divs are all part of the individual dots making up the loader